user_sync/connector/directory_ldap.py

```diff
@@ -197,6 +232,32 @@ def find_ldap_group_dn(self, group):
                 group_dn = current_tuple[0]
         return group_dn
 
+    def find_group_member_dn(self, group, member_attribute):
+        """
+        :type group: str
+        :type member_attribute: str
+        :rtype list
+        """
+        connection = self.connection
+        options = self.options
+        base_dn = six.text_type(options['base_dn'])
+        group_filter_format = six.text_type(options['group_filter_format'])
+        try:
+            filter_string = self.format_ldap_query_string(group_filter_format, group=group)
+            res = connection.search_s(base_dn, ldap.SCOPE_SUBTREE,
+                                      filterstr=filter_string, attrlist=[member_attribute])
+        except Exception as e:
+            raise AssertionException('Unexpected LDAP failure reading group info: %s' % e)
+        group_dn = None
+        for current_tuple in res:
+            if current_tuple[0]:
+                if group_dn:
+                    raise AssertionException("Multiple LDAP groups found for: %s" % group)
+                group_dn = current_tuple[0]
+                if member_attribute in current_tuple[1]:
+                    for member in current_tuple[1][member_attribute]:
+                        yield member
```

Comment:
Aha!  This is actually an iterator!  The name should be called `iter_group_member_dns` and it has no `rtype`.